pipeline-model-definition/src/test/java/org/jenkinsci/plugins/pipeline/modeldefinition/AbstractModelDefTest.java

```diff
@@ -596,10 +596,10 @@ public WorkflowRun getRun() {
     }
 
     public class EnvBuilder {
-        private final Slave agent;
```

Comment:
This part is code change and should be reverted too.

Reply:
Agreed.  This needs to be reverted